package.json

```diff
@@ -0,0 +1,337 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack serve --mode development --open",
+    "build": "webpack --mode production",
+    "start": "node server.js"
```

Comment:
- [ ] I see that you created a `server.js` file, but this is not an Express.js application and the `npm start` command is not working. Kindly [read this documentation](https://webpack.js.org/guides/development/), and make sure all the scripts are working(`dev`, `build`, `start`).